src/scripts/modules/media/body/body.coffee

```diff
@@ -409,3 +417,10 @@ define (require) ->
 
     resetKeySequence: (e) ->
       key[e.keyCode] = false
+
+    simLink: (evt) ->
+      evt.preventDefault()
+      link = $(evt.currentTarget)
+      @model.set('simUrl', link.attr('href'))
```

Comment:
Oh, I thought the sims would still be an `<iframe>`. When did `ost-iframe-embeddable` get defined? I don't see it defined anywhere in https://github.com/openstax/tagging-legend/blob/master/README.md